src/test/java/org/openrefine/extensions/commons/importer/FileRecordToRowsTest.java

```diff
@@ -26,9 +26,13 @@ public void testGetNextRowOfCells() throws Exception {
         List<Object> rows = new ArrayList<>();
         rows.add(frtr.getNextRowOfCells());
         rows.add(frtr.getNextRowOfCells());
+        rows.add(frtr.getNextRowOfCells());
+        rows.add(frtr.getNextRowOfCells());
```

Comment:
At the end you could also check that `getNextRowOfCells()` returns `null` (which signals the end of the table).